UnitTesting-master/src/test/java/testNG/JsonParserTestNG.java

```diff
@@ -0,0 +1,79 @@
+package testNG;
+
+import com.github.javafaker.Faker;
+import com.google.gson.Gson;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+import parser.JsonParser;
+import parser.NoSuchFileException;
+import shop.Cart;
+import shop.RealItem;
+import shop.VirtualItem;
+
+import java.io.*;
+
+import static org.testng.Assert.*;
+
+public class JsonParserTestNG {
+
+    private final JsonParser jsonParser = new JsonParser();
+
+    private static final Faker faker = new Faker();
+    private static final String funnyCartName = faker.funnyName().name();
+    private static final String funnyCartName2 = faker.funnyName().name();
+    private final Cart cart = new Cart(funnyCartName);
+    private final Cart cart2 = new Cart(funnyCartName2);
```

Comment:
You can have 1 Cart object and initialize it in method annotated with @BeforeMethod